app/templates/components/task-item.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="card">
+  <header class="card-header">
+    <p class="card-header-title">
+      {{task.title}}
+    </p>
+    <a href="{{task.url}}" target="_blank" class="card-header-icon" aria-label="more options">
+      <span class="icon">
+        {{#if task.isPullRequest}}
+          {{fa-icon 'code-branch'}}
+        {{else}}
+          {{fa-icon 'bug'}}
+        {{/if}}
+      </span>
+    </a>
+  </header>
+  <div class="card-content">
+    <div class="content">
+      {{truncate-text task.bodyText limit=350}}
+    </div>
+    <time title="{{task.updatedAt}}">{{time-ago task.updatedAt}}</time>
+  </div>
+  <footer class="card-footer">
+    <a href="{{task.repository.url}}" class="card-footer-item" target="_blank">
+      <figure class="org-icon">
+        <img src="{{task.repository.owner.avatarUrl}}" alt="">
+      </figure>
+      &nbsp;
+      {{task.repository.nameWithOwner}}
```

Comment:
As all items in the list are in the same owner, and the owner avatar is shown, IMO the owner name shouldnt be shown here as it is redundant.

Reply:
If redundant is your concern. I will remove the avatar part as it is not and Identifiable object.

Reply:
It is redundant because every task in the list has the same org.
That is not fixed by removing the avatar.

Reply:
I think he meant the `coala` part shouldn't be shown because it'll always be `coala` in current list.

Reply:
I think he meant the `coala` part shouldn't be shown because it'll always be `coala` in current list.